### PR TITLE
SW-5659: FE - Application success screen

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR includes changes to add the application status submit success screen.

## Screenshots

localhost_3000_applications_1_review_organizationId=141

### CODE REVIEW

File: src/scenes/ApplicationRouter/portal/Review/index.tsx [hunk 1]
```diff
@@ -9,6 +13,50 @@ import strings from 'src/strings';
...
+  );
+};
+
+const ApplicationStatusSubmitted = () => (
+  <ApplicationStatus
+    buttonLabel={strings.EXIT_APPLICATION}
+    feedback='<p>Your Application has been submitted to the Accelerator team. We will review and score your answers.</p><p>Check back to see updates? Look for an email? [need copy]</p>'
+    isFailure={false}
+    onClickButton={() => {
+      // TODO: what does "Exit Application" do?
```
Comment (tommylau523) [inline src/scenes/ApplicationRouter/portal/Review/index.tsx:54]: Exit application just brings us back to home page. I have been using goToHome from useNavigateTo

File: src/scenes/ApplicationRouter/portal/Review/index.tsx [hunk 2]
```diff
@@ -32,6 +80,7 @@ const ReviewView = () => {
 
   return (
     <ApplicationPage crumbs={crumbs}>
+      {selectedApplication.status === 'Submitted' && <ApplicationStatusSubmitted />}
```
Comment (tommylau523) [inline src/scenes/ApplicationRouter/portal/Review/index.tsx:83]: We can change this to 'In Review' instead. Submitted is an internal one.

The list of statuses are here:
https://terraformation.atlassian.net/wiki/spaces/PM/pages/1295319074/PRD+Application+-+Pre-screen#Data-%2F-Calculations

Comment (tommylau523) [reply src/scenes/ApplicationRouter/portal/Review/index.tsx:83]: Adding on to this, I think we will need slightly different text/screens for Accepted, Waitlist, Not Accepted as well. But we can follow up with more PRs instead.

**Review by tommylau523 — APPROVED** [review]
Looks great! I think we need to rework the status to be a switch case at some point to make sure we don't miss any statuses. But this can happen later.

**Comment by joshbuchea** [comment]
@tommylau523 thanks for the feedback! 🙌 I agree that reworking the code to utilize a switch case is a good idea. Will get the requested changes implemented.

**Pull request merged**